src/css/styles.css

```
@@ -111,9 +111,9 @@ html {
 }
 
 .district {
-  font-weight: 300;
-  color: #000;
+  color: #777;
```

The text element `font-weight` is light (300), using a font color this light will make the color contrast not sufficient enough for a white background. Let's use a bit darker color like `#595959`. 

See https://dequeuniversity.com/rules/axe/3.1/color-contrast

```suggestion
  color: #595959;
```